test/shorthands.jl

```diff
@@ -0,0 +1,126 @@
+video = Video(800, 800)
+
+@testset "Shorthands" begin
```

Comment:
Either in this test set or in the Layer test set, could you test running J-Objects within a Layer?

Reply:
done in the Layer tests with JShape.